src/rmodels.c

```diff
@@ -5137,18 +5129,18 @@ static Model LoadGLTF(const char *fileName)
                             if (attribute->component_type == cgltf_component_type_r_32f)  // vec2, float
                             {
                                 // Init raylib mesh texcoords to copy glTF attribute data
-                                texcoordPtr = RL_MALLOC(attribute->count*2*sizeof(float));
+                                texcoordPtr = (float *)RL_MALLOC(attribute->count*2*sizeof(float));
```

Comment:
Those changes are already on raylib master branch, they shouldn't be in this changelist.